ddtrace/internal/coverage/instrumentation_py3_10.py

```diff
@@ -0,0 +1,285 @@
+import sys
+
+
+if sys.version_info >= (3, 10) and sys.version_info < (3, 11):
```

Comment:
can we remove these checks?

Reply:
Swapped them for asserts (I don't love it, but it's the only other option I've found).